platform/linux-generic/odp_schedule_basic.c

```diff
@@ -41,15 +42,18 @@ ODP_STATIC_ASSERT((ODP_SCHED_PRIO_NORMAL > 0) &&
 /* Number of scheduling groups */
 #define NUM_SCHED_GRPS 32
 
-/* Priority queues per priority */
-#define QUEUES_PER_PRIO  4
+/* Maximum priority queue spread */
+#define MAX_SPREAD 4
```

Comment:
This limit seems to contradict the recommendation mentioned earlier.

Reply:
On this patch level, max is still 4. A later patch increases it to 8.